plugin/evm/vm.go

```diff
@@ -480,6 +483,20 @@ func (vm *VM) Initialize(
 		}
 	}
 
+	// parse and add valid off-chain warp messages to warp backend
+	for _, message := range vm.config.OffChainWarpMessages {
+		messageByte, err := hex.DecodeString(message)
+		if err != nil {
+			return errInvalidOffChainWarpMessage
+		}
+		unsignedMessage, err := avalancheWarp.ParseUnsignedMessage(messageByte)
+		if err != nil {
+			return errInvalidOffChainWarpMessage
+		}
```

Comment:
Can we change from doing this here to adding an argument to the constructor of the warp backend that takes in a slice of byte arrays to add as the set of hardcoded off-chain warp messages?

Inside the warp backend, we should also parse the unsigned message and parse the underlying payload as well to ensure that it's valid.